TREKPOLOGY/src/styles/client.less

```
@@ -14754,3 +14754,433 @@ body,
+
+.framed-card-face__name,
+.framed-card-face__vp,
+.framed-card-face__pill,
+.framed-card-face__cost,
+.framed-card-face__description {
+  position: absolute;
+  z-index: 2;
+  margin: 0;
+  color: #354724;
+  font-family: inherit;
+  text-rendering: geometricPrecision;
```

Stylelint (17.13.0)

[error] Expected "geometricPrecision" to be "geometricprecision" (value-keyword-case)

(value-keyword-case)